urlshort/handler.go

```diff
@@ -12,7 +14,14 @@ import (
 // http.Handler will be called instead.
 func MapHandler(pathsToUrls map[string]string, fallback http.Handler) http.HandlerFunc {
 	//	TODO: Implement this...
-	return nil
+	return func(w http.ResponseWriter, r *http.Request) {
+		path, ok := pathsToUrls[r.URL.Path]
+		if ok {
+			http.Redirect(w, r, path, http.StatusFound)
+		} else {
+			fallback.ServeHTTP(w, r)
+		}
```

Comment:
In go you will typically return instead of using else statements where appropriate.
```suggestion
		if path, ok := pathsToUrls[r.URL.Path]; ok {
			http.Redirect(w, r, path, http.StatusFound)
		}
		
		fallback.ServeHTTP(w, r)
```